README.md

```diff
@@ -7,11 +7,16 @@
 - `make`
 - **[go 1.10](https://golang.org/dl/)**
 - **protobuf 3.2**: `brew install protobuf` or install `protoc-3.2.0-[your_OS]` [from Github](https://github.com/google/protobuf/releases?after=v3.2.1)
+- Our Python scripts are in Python 3.6.0. To run them:
+  - **[pyenv](https://github.com/pyenv/pyenv#installation)**
+  - **python**: `pyenv install`
+  - if you're using GKE and get the error: "pyenv: python2: command not found", run:
```

Comment:
fyi @nicks @jazzdan this is a potentially awkward result of having our scripts be in py3. I've got stuff working fine now, but if we think this is too much of a PITA for ppl trying to run the scripts, we can easily change them back to py2. thoughts?